Gemfile.lock

```diff
@@ -812,4 +821,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.1.4
+   2.2.32
```

Comment:
I don't think this change is necessary/wanted.

Reply:
Oops, good pickup thanks! Updated the commit.